tests/projects/test_cavatica.py

```diff
@@ -35,13 +35,9 @@ def test_correct_projects(db, mock_create_project):
 
     setup_cavatica(study)
 
-    assert mock_create_project.call_count == 3
+    assert mock_create_project.call_count == 1
```

Comment:
Could you add another test just like this that overrides the default projects and asserts that they are created?